test/common/common.go

```diff
@@ -50,9 +51,11 @@ const (
 
 	// jqExtractRegistryLocation is a jq command that extracts the source location of the '/var/lib/registry' mount from the container's config.json file.
 	jqExtractRegistryLocation = `jq -j '.mounts[] | select(.destination=="/var/lib/registry") | .source' /run/containerd/io.containerd.runtime.v2.task/k8s.io/%s/config.json`
-	// jqCountManifests is a jq command that counts the number of image manifests in the manifest index.
-	// Ref: https://github.com/opencontainers/image-spec/blob/main/image-index.md#example-image-index.
-	jqCountManifests = `jq -j '.manifests | length' %s/docker/registry/v2/blobs/%s/data`
+	// jqCurrentManifestDigest is a jq command that extracts the manifest digest for the current OS architecture
+	jqCurrentManifestDigest = `jq -j '.manifests[] | select(.platform.architecture=="%s") | .digest' %s/docker/registry/v2/blobs/%s/data`
```

Comment:
I would name it something like `jqExtractManifestDigest` - I don't know what current is supposed to be in the current name; additionally I am missing a verb that describes the operation.